OptimizelySDK/AudienceConditions/BaseCondition.cs

```diff
@@ -156,6 +170,35 @@ public class BaseCondition : ICondition
             return Convert.ToDouble(attributeValue) > Convert.ToDouble(Value);
         }
 
+        public bool? GreaterOrEqualThanEvaluator(object attributeValue, ILogger logger)
```

Comment:
Am not in favor of all these evaluator or we should put common logic in a separate method, the only evaluation logic should be kept inside every evaluator.

Reply:
Lets talk offline. CommonEvaluator should return either 0, 1, -1 or raise exception. We shouldn't parse anything in GE, LE, EQ (SemVer).